modules/miner/miner.go

```diff
@@ -149,6 +168,23 @@ func New(cs modules.ConsensusSet, tpool modules.TransactionPool, w modules.Walle
 		arbDataMem: make(map[types.BlockHeader][crypto.EntropySize]byte),
 		headerMem:  make([]types.BlockHeader, HeaderMemory),
 
+		fullSets:  make(map[modules.TransactionSetID][]int),
+		splitSets: make(map[splitSetID]*splitSet),
+
+		blockMapHeap: &mapHeap{
```

Comment:
no space here - when initializing structs we try to match the spacing of their definitions.